### PR TITLE
HP Cloud sucks, so we have to reboot to avoid wasting ALL our time

### PR DESCRIPTION
**Pull request by sorenh**

### CODE REVIEW

File: build_scripts/deploy.sh [hunk 1]
```diff
@@ -12,6 +12,10 @@ fi
 
 time python -m jiocloud.apply_resources apply ${EXTRA_APPLY_RESOURCES_OPTS} --key_name=${KEY_NAME:-combo} --project_tag=${project_tag} ${mappings_arg} environment/${layout}.yaml userdata.txt
 
+# This is crazy, but it seems to help A LOT
```
Comment (bodepd) [inline build_scripts/deploy.sh]: would you mind putting this in a if cloud_provider = hp block?

Comment (sorenh) [inline build_scripts/deploy.sh]: What a wonderful idea. I'll get right on that.

**Comment by bodepd** [comment]
please test this

**Comment by sorenh** [comment]
please test this

**Pull request merged**